lib/resources/postgres.rb

```diff
@@ -36,6 +36,9 @@ def initialize
         version = if inspec.directory('/var/lib/pgsql/data').exist?
                     warn 'Found /var/lib/pgsql/data. Assuming postgresql install uses un-versioned directories.'
                     nil
+                  elsif inspec.directory('/var/lib/postgresql/data').exist?
```

Comment:
I'm not sure.  Now that you mention it, I assume it is OS specific.  It is Alpine-based.
https://github.com/docker-library/postgres/blob/3d4e5e9f64124b72aa80f80e2635aff0545988c6/9.4/alpine/Dockerfile